src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
```

Comment:
It's a good practice to use even numbers for margins, paddings and other measurements in CSS. This makes the design more consistent. Consider changing margin-left to 4px.